shared/js/smallFooter.js

```diff
@@ -0,0 +1,111 @@
+/* global $ */
```

Comment:
It's provided by enabling the JSHint [browser option](http://jshint.com/docs/options/#browser) in apps/Gruntfile.js